### PR TITLE
Updates for sarek 3.4.2

### PR DESCRIPTION
**Pull request by aanil**

### CODE REVIEW

File: ngi_pipeline/engines/sarek/models/sarek.py [hunk 1]
```diff
@@ -529,9 +529,10 @@ def generate_tsv_file_contents(self, analysis_sample):
         """
         rows = []
         patientid = analysis_sample.sampleid
-        gender = "ZZ"
+        gender = "NA"
         status = 0
         sampleid = patientid
+        rows.append(['patient','sex','status','sample','lane','fastq_1','fastq_2'])
```
Comment (FranBonath) [inline ngi_pipeline/engines/sarek/models/sarek.py:535]: do you need to remove something when you add this?

**Review by FranBonath — APPROVED** [review]

File: ngi_pipeline/engines/sarek/models/sarek.py (hunk 1 above)
Comment (aanil) [reply ngi_pipeline/engines/sarek/models/sarek.py:535]: The earlier input tsv file didn't have headers. The latest sarek expects them

Comment (FranBonath) [reply ngi_pipeline/engines/sarek/models/sarek.py:535]: "the earlier input tsv file didn't have headers. The latest sarek expects them"

Comment (FranBonath) [reply ngi_pipeline/engines/sarek/models/sarek.py:535]: lol

**Pull request merged**